models/models.reviews.js

```diff
@@ -0,0 +1,11 @@
+const db=require('../db/connection')
+
+exports.checkIfIDExists=(id)=>{
```

Comment:
I do like the idea of a utils function here to check if the ID exist. But I would suggest that it isn't strictly necessary. We could simply run the valid but non existent ID through our SQL query in the standard model and check the rowCount of the response. If 0, reject it and catch it with your custom error handler. You are essentially doing the same thing here so it's repeated code.